frontend/src/app/modules/common/highlight-col/highlight-col.directive.ts

```diff
@@ -24,7 +24,7 @@
 // Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 //
 // See docs/COPYRIGHT.rdoc for more details.
-// ++
+//++
 
 import {Component, ElementRef, OnInit, OnDestroy} from '@angular/core';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/2bb05fb5a11e46adae40e74960f14137703ba57b/frontend/src/app/modules/common/highlight-col/highlight-col.directive.ts#L30"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>